kzg_prover/src/circuits/tests.rs

```diff
@@ -122,39 +122,64 @@ mod test {
         // Both the Custodian and the Verifier know what column range are the balance columns
```

Comment:
Is `poly_degree` the same as number of users?

Reply:
No, it can be different because hypothetically there can be something else in the circuit, or simply the number of users is not a power of two.

Reply:
Actually they are not the same right now because K = 9 so there are 2^9 points that the advice polynomials interpolate, but only 16 of them are filled with the user balances during the test.